frontend/packages/noobaa-storage-plugin/src/components/object-bucket-claim-page/modals/attach-deployment-obc-modal.tsx

```diff
@@ -81,23 +73,23 @@ const AttachDeploymentToOBCModal = withHandlePromise((props: AttachDeploymentToO
       if (_.isEmpty(container.env)) {
```

Comment:
Shouldn't this be checking `envFrom`?

```suggestion
      if (_.isEmpty(container.envFrom)) {
```